front/app/modules/commercial/content_builder/admin/components/RenderNode/index.tsx

```diff
@@ -19,13 +19,15 @@ const TWO_COLUMNS = 'TwoColumn';
 const THREE_COLUMNS = 'ThreeColumn';
 const TEXT = 'Text';
 const IMAGE = 'Image';
+const ABOUTBOX = 'AboutBox';
 
 type ComponentNamesType =
   | typeof CONTAINER
   | typeof TWO_COLUMNS
   | typeof THREE_COLUMNS
   | typeof TEXT
-  | typeof IMAGE;
+  | typeof IMAGE
+  | typeof ABOUTBOX;
```

Comment:
Super nitpicky but maybe you can keep the name consistent with the column components `ABOUT_BOX`?